src/Form/matching-configuration.js

```diff
@@ -274,11 +268,34 @@ const matchingConfiguration = {
                 },
 
                 // Identities
-                firstName: {match: '(first|given|fore).?name', skip: 'last'},
-                middleName: {match: '(middle|additional).?name'},
-                lastName: {match: '(last|family|sur)[^i]?name', skip: 'first'},
-                fullName: {match: '^(full.?|whole\\s|first.*last\\s|real\\s|contact.?)?name\\b', forceUnknown: 'company|org|item'},
-                phone: {match: 'phone', skip: 'code|pass|country', forceUnknown: 'ext|type|otp'},
+                firstName: {
+                    match: '(first|given|fore).?name' +
+                        // Italian
+                        '|\\bnome',
```

Comment:
Translating identities is out of scope, but I've dropped these in for now. I may decide to remove them, or to add translations for everything.